install/helm/agones/templates/controller.yaml

```diff
@@ -81,3 +81,7 @@ spec:
       - name: certs
         secret:
           secretName: {{ template "agones.fullname" . }}-cert
+{{- if .Values.agones.image.controller.pullSecret }}
```

Comment:
Just thought of one more thing - this should be documented here:
https://github.com/GoogleCloudPlatform/agones/blob/master/install/helm/README.md#configuration

Which I think will solve #358's issue as well - won't it?

Reply:
Yes, indeed.